docs/devguide/modules/install-sbo/partials/minikube/helm-install.adoc

```diff
@@ -0,0 +1,40 @@
+== Install with Helm
+
+1. Build the operator's image
++
+[source,bash]
+----
+make image
+----
+
+2. Install the CRDs
++
+[source,bash]
+----
+make install
+----
+
+3. Deploy the operator using Helm
++
+[source,bash]
+----
+helm install \
+    --create-namespace \
+    --namespace service-binding-operator \
+    --set image.repository=$OPERATOR_IMAGE_REF \
```

Comment:
See the https://github.com/redhat-developer/service-binding-operator/pull/1309#discussion_r1030395699